src/debugger/include/debugger/debugger_interface.h

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file debugger_interface.h
+ * Declares the DebuggerCore Interface Class for CLI Operations
```

Comment:
```suggestion
 * Declares the Debugger Interface Class
```

This is not exclusively for CLI operations, no?